content/en/docs/concepts/security/pod-security-standards.md

```diff
@@ -76,7 +76,7 @@ fail validation.
 		<tr>
 			<td style="white-space: nowrap">Host Namespaces</td>
 			<td>
-				<p>Sharing the host namespaces must be disallowed.</p>
+				<p>Sharing the host namespaces must be disallowed. This is a Linux specific field and should not be used for Windows Pods.</p>
```

Comment:
does it make sense to mention the fact that this will be effectively only possible with the new Windows KEP being in place?

Reply:
we would not add OS-specific docs to this check... no matter the OS of the pod, this policy would disallow this field

Reply:
So, what should the admission plugin do? Throw an error saying:

`This field is not allowed when baseline policy is enabled`?

or

`This field is not allowed for Windows pod and cannot be used with baseline profile even for linux pods` to give user a more specific error?

or are you saying let's not update the docs for OS specific fields if they are already validated based on values(sometimes nil or undefined etc).

Reply:
All the checks that forbid specific fields/values can remain unchanged, even if those fields are OS-specific. For example, the baseline policy can forbid `privileged: true`, regardless of whether the pod is a windows pod or not.

(API validation could additionally complain if you set a linux-specific field on an explicitly windows pod, but that would not be done by the pod security admission plugin)

The checks that currently *require* specific fields to be set (which are all in the `restricted` policy) are what we would make conditional on the pod OS.

Reply:
> (API validation could additionally complain if you set a linux-specific field on an explicitly windows pod, but that would not be done by the pod security admission plugin)

I am doing it here - https://github.com/kubernetes/kubernetes/pull/104693

> The checks that currently require specific fields to be set (which are all in the restricted policy) are what we would make conditional on the pod OS.

Ok, I'll make changes to restricted profile only then.